routingpy/routers/google.py

```diff
@@ -189,8 +187,8 @@ def directions(  # noqa: C901
         language: Optional[str] = None,
         region: Optional[str] = None,
         units: Optional[str] = None,
-        arrival_time: Optional[int] = None,
-        departure_time: Optional[int] = None,
+        date_time: Optional[datetime.datetime] = datetime.datetime.now(datetime.timezone.utc),
```

Comment:
I made them all more consistent in terms of time arguments:
- `date_time` is everywhere supposed to be a tz-aware date_time object, defaulting to UTC
- `date_time_type` is either `depart_at` or `arrive_by`

I put those arguments wherever time is needed, but sometimes they're not doing anything, e.g. OTP isochrones only support `depart_at` scenarios

Reply:
We could've done a enum or so, but I thought with a docstring it's ok, and it's one less thing to import for others

Reply:
Great, I'd planned to tackle that later too! However, I would have preferred a "departure_datetime" and an "arrival_datetime". I find it more concise and more explicit for the user.

Reply:
I'm open to that. As we're breaking stuff right now, we're flexible. And I kinda agree.

Reply:
I'll change it here before touching the tests.